tools/build

```diff
@@ -77,6 +86,16 @@ while [ $# -gt 0 ]; do
             shift
             ;;
 
+        --cargo-pgx)
```

Comment:
Either we need to change the -pg* options to use double-dash or change these two to use single. I prefer the latter, but won't put up a fight over it. As long as we're consistent.

Reply:
I don't have any taste when it comes to shell scripts. Does single-dash mean we have to do `-cargopgx` and `-cargopgxold`? Or is `-cargo-pgx-old` possible?

Reply:
`-cargo-pgx-old` is fine. Feel free to adjust the others for consistency. `-pgconfig` looks fine but `-cargopgxold` is definitely too many words smooshed together :)